### PR TITLE
Use HTTPS/SSL

### PR DESCRIPTION
**Pull request by rubenmoya**

### CODE REVIEW

File: load_balancer.tf [hunk 1]
```diff
@@ -29,16 +29,34 @@ resource "aws_lb" "public_lb" {
 
 resource "aws_lb_target_group" "ec2_tg" {
   name     = "Terraform-EC2-TargetGroup"
-  port     = 80
-  protocol = "HTTP"
+  port     = 443
```
Comment (ghost) [inline load_balancer.tf]: This change is not necessary. The target group specifies to which port of the receiving instances (in our case the Swarm Instances) will the requested be forwarded to. In our case we want the instances to listen to http on port 80 and let the LB handle the SSL

File: load_balancer.tf [hunk 2]
```diff
@@ -29,16 +29,34 @@ resource "aws_lb" "public_lb" {
...
+      status_code = "HTTP_301"
+    }
+  }
+}
+
+resource "aws_lb_listener" "port_443_listener" {
+  load_balancer_arn = "${aws_lb.public_lb.arn}"
+  port              = 443
+  protocol          = "HTTPS"
+  ssl_policy        = "TODO"
```
Comment (ghost) [inline load_balancer.tf]: Searching in the [AWS docs](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/create-https-listener.html#describe-ssl-policies) they suggest using "ELBSecurityPolicy-2016-08. 
```suggestion
  ssl_policy        = "ELBSecurityPolicy-2016-08"
```

File: load_balancer.tf [hunk 3]
```diff
@@ -29,16 +29,34 @@ resource "aws_lb" "public_lb" {
...
+  certificate_arn   = "TODO"
```
Comment (ghost) [inline load_balancer.tf]: For setting this up we could upload the SSL certificate in the [Certificate Manager](https://eu-west-1.console.aws.amazon.com/acm/home?region=eu-west-1#/) through the AWS Dashboard, and then use a [Data source](https://www.terraform.io/docs/providers/aws/d/acm_certificate.html) to reference the Certificate inside terraform. For more info on Data Sources see [this](https://www.terraform.io/docs/configuration/data-sources.html).

**Review by ghost — CHANGES_REQUESTED** [review]

File: variables.tf [hunk 4]
```diff
@@ -1,5 +1,14 @@
 data "aws_availability_zones" "available" {}
 
+data "aws_acm_certificate" "default" {
+  domain      = "${var.site_name}"
+  most_recent = true
+}
+
+variable "site_name" {
```
Comment (rubenmoya) [inline variables.tf:8]: I'm not sure about keeping this since it only adds a description... what do you think @GustavoTango?

**Review by ghost — APPROVED** [review]

**Pull request merged**